ratelimitj-core/src/test/java/es/moki/ratelimitj/core/limiter/request/RequestLimitRuleTest.java

```diff
@@ -59,7 +66,23 @@ void shouldHaveLimitGreaterThanZero() {
 
     @Test
     void shouldHaveDurationGreaterThanOneSecond() {
-        assertThatThrownBy(() -> RequestLimitRule.of(Duration.ofMillis(100), 1).withName("boom")).isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> RequestLimitRule.of(Duration.ofMillis(100), 1).withName("boom"))
```

Comment:
Adding more tests about exception and validating the message to assure it is the right exception.
I'm assuming we don't want different exception classes, for each invalid time field